src/components/Tree/Tree.spec.tsx

```diff
@@ -0,0 +1,97 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconSize } from "@components/Icon/Icon";
+import { ReactComponent as IconDocument } from "@components/Icon/Svg/Document.svg";
+import { ReactComponent as IconGuidelines } from "@components/Icon/Svg/Guidelines.svg";
+import { mount } from "@cypress/react";
+import { FC, useState } from "react";
+import { TreeNode } from "./Node";
+import Tree from "./Tree";
+import css from "./Tree.module.css";
+
+const Tree_FOO = "bar";
+
+const Component: FC<{ nodes: TreeNode[] }> = ({ nodes }) => {
+    const [selectedId, setSelectedId] = useState<string | undefined>();
+    return <Tree nodes={nodes} selectedNodeId={selectedId} onSelect={(id: string) => setSelectedId(id)} />;
+};
+
+const TREE_ID = "[data-test-id=Tree]";
```

Comment:
we agreed on using kebap-case for `data-test-id`s

Reply:
952bc7042225491892aaccd9af3d4533304d36b3